ocpp-jaxb/README.md

```
@@ -0,0 +1,41 @@
+    ...
+    <repository>
+        <id>JitPack</id>
+        <url>https://jitpack.io</url>
+    </repository>
+    ...
+</repositories>
+ ```
+
+Then, add the following to your pom.xml (use any version from [github tags page](https://github.com/RWTH-i5-IDSG/ocpp-jaxb/tags)):
+
+ ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)